pkg/minikube/cruntime/docker.go

```diff
@@ -106,8 +106,17 @@ func (r *Docker) SocketPath() string {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *Docker) Available() error {
-	_, err := exec.LookPath("docker")
-	return err
+	var err error
+	if _, err = exec.LookPath("docker"); err != nil {
+		return err
+	}
+	if _, err = exec.LookPath("cri-dockerd"); err == nil {
+		return nil
+	}
+	if _, err = exec.LookPath("dockerd"); err == nil {
```

Comment:
It looks like that if `docker` and `cri-dockerd` are installed the check for `dockerd` doesn't occur, but we need to make sure all three of them are there.

Reply:
Thanks for taking a look, now ensuring that all three are there.